docs/android.md

```diff
@@ -71,3 +71,51 @@ Dependencies tab.
 Caution: adding dependencies also adds code to your app.
 This increases the size of the installed binary on user's
 devices. The appcompat-v7 library adds almost 7 megabytes.
+
+## Plugins
+
+Editing the native Android code of a Flutter plugin is a bit
+more tricky. Let's say we have created a new plugin called `fpl`.
+It might have been created using an IDE or perhaps it was made
+at the command line:
+
+    flutter create --template=plugin fpl
+
+To edit the native Android code you need to follow these steps
+the first time, to get dependencies and paths configured.
+1. Open `fpl` as the top-level project in Android Studio or IntelliJ.
+2. Use the device picker to start the Android emulator.
+3. Run the default run configuration, which launches the `example`
+   app on the emulated device. This causes Flutter to build all
+   the dependent binaries and get the paths set up correctly,
+   the Flutter equivalent to doing a Gradle build.
+
+    Alternatively, if you have an Android device connected (or
+    an emulator running) these first three steps could be done
+    from the command line.
+
+        cd fpl/example
+        fluter run
```

Comment:
sp: flutter

Reply:
I'd watch this after the PR lands to see if the markdown formatter actually puts these on separate lines.

Reply:
SGTM. The markdown preview in IntelliJ does it but github may be different.

Reply:
Looks like github does the right thing here.